Jenkinsfile

```diff
@@ -134,7 +136,7 @@ dockerizedBuildPipeline(
   testResults: ['lib/junit.xml', 'gallery/test-results/junit.xml'],
   onSuccess: {
     githubStatusUpdate('success')
-    if (env.BRANCH_NAME == 'main') {
+    if (env.BRANCH_NAME == deployBranch) {
```

Comment:
This one shouldn't have been changed I don't think.  I assume it will mess up gallery.sonatype.dev if a release gets deployed there that isn't the highest

Reply:
> This one shouldn't have been changed I don't think. I assume it will mess up gallery.sonatype.dev if a release gets deployed there that isn't the highest

That makes sense: I've changed it back here: 5a7240af00f8658e2d14cf5c79549e90a1773455